ethbloom/src/lib.rs

```diff
@@ -140,13 +140,13 @@ impl Bloom {
 	}
 
 	pub fn contains_bloom<'a, B>(&self, bloom: B) -> bool where BloomRef<'a>: From<B> {
-		let bloom_ref: BloomRef = bloom.into();
+		let bloom_ref: BloomRef<'_> = bloom.into();
```

Comment:
are all of them needed or just automatically made by a tool ?

Reply:
It fixes a warning of `rust 2018 idioms` (warning: hidden lifetime parameters in types are deprecated),  see https://github.com/rust-lang/rust/issues/45992

I think @DemiMarie-parity has executed `cargo fix --edition-idioms`, right?

Reply:
@niklasad1 correct.